sci-physics/yoda/Manifest

```diff
@@ -1,7 +1,2 @@
-DIST YODA-1.9.0.tar.bz2 1061105 BLAKE2B 70d68d31456747fcb14e27585aae245930061b5135b8a96c8ff3fe0375175d00a1c2d24a6c1ca3f18fc8464e610ef6287184f3990e12ff3fb108def600e34189 SHA512 eb82f1df832c2124dd81e07c06f165ddf13c4c4b70fcd0113f85045e22085d82751325bc57bfbd167649580980d98a15b666de56dada639160f24682b58b2f71
 DIST YODA-1.9.10.tar.bz2 1278729 BLAKE2B f6ae9b579d79b6be62e9644e327f0b449882c0f1a0efaac09ac97eae90a2ced10bf2b13447d75390552b3e4d8c128e7a33184bf8b2d308236f828e611df6d7a4 SHA512 54a455a1b2772ea3201477d4dddcf0c92c85bfee8825d8726d62331ebacb37fc8d1f6e02cdfe592722f2a747a46786d2bdb3c36cc982794a30b25e718d32e998
-DIST YODA-1.9.5.tar.bz2 1077442 BLAKE2B 010ab725c57dfab8160e0304d461dc44c140d1ce234f3b70834ee58b8f269159529122095a94e6b79ef752612a6659a17e1874fda2941cfbcec1f53802916b6a SHA512 46e30b33d369f5d28ec228b71eaa298f3010cc06fe568785c74bbaf3d7468c26738cc2ee40da9bcc54d978f394187a2ee9289197283c86a3358583a3c0ecd247
```

Comment:
We should keep 1.9.5 for now since it is the last version with x86 keywords (The QA bot should have complained about this, but somehow did not).

We should also open a rekeywording request for the newer versions if we have not done so already.

Reply:
I have created one here: https://bugs.gentoo.org/941638